ibis/__init__.py

```diff
@@ -20,6 +20,8 @@
 
 __version__ = "3.1.0"
 
+KNOWN_BACKENDS = ['bigquery', 'heavyai']
```

Comment:
There's `mssql` and `omniscidb` (former name for `heavyai`) although I'm not sure if those backends are up-to-date with ibis 3.x

Reply:
They likely aren't but I think that's okay.

Reply:
Is it worth hiding this as `_KNOWN_BACKENDS`?

Reply:
ok, it looks like `ibis-omniscidb` hasn't been updates, so skip that one.  checking on `ibis-mssql`

Reply:
I'd say so, we don't want to export this as public API.

Reply:
yeah, and `ibis-mssql` hasn't been updated in 18 months, ok, ignore this!